internal/handler/messaging.go

```diff
@@ -192,10 +206,22 @@ func (h *Messaging) processMessageQueue(message mq.Message) {
 			err := h.sendToLagoonAPI(incoming, resource, insights)
 
 			if err != nil {
-				//log.Printf("Unable to send to the api: %s", err.Error())
-				slog.Error("Unable to send to the API", "Error", err.Error())
-				rejectMessage(false)
-				return
+				incoming.RequeueAttempts++
```

Comment:
So, here's a thought - what if we moved this repeated rejection/retry logic into the "rejectMessage" function itself?

That function has access to all this information, as well as having the ability to requeue?